recipes/yoga/all/patches/0001-delete-tests.patch

```diff
@@ -2,11 +2,8 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index f4ce73cc..9f414127 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -10,7 +10,6 @@ set(CMAKE_VERBOSE_MAKEFILE on)
+@@ -10,4 +10,3 @@ set(CMAKE_VERBOSE_MAKEFILE on)
  include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/project-defaults.cmake)
  
  add_subdirectory(yoga)
 -add_subdirectory(tests)
```

Comment:
I'd recommend opening an issue and maybe even MR upstream to add an option to avoid building tests for the project altogether. Then this patching won't be necessary.

Reply:
Good idea, but I have to check first whether my company allows the CLA

Reply:
Thanks!